### PR TITLE
Add CovidQA dataset

### PR DESCRIPTION
**Pull request by olinguyen**
This PR adds CovidQA, a question answering dataset specifically designed for COVID-19, built by hand from knowledge gathered from Kaggle’s COVID-19 Open Research Dataset Challenge.

Link to the paper: https://arxiv.org/pdf/2004.11339.pdf
Link to the homepage: https://covidqa.ai

### CODE REVIEW

**Comment by xhlulu** [comment]
It seems to share the same name as this dataset: https://openreview.net/forum?id=JENSKEEzsoU

**Comment by olinguyen** [comment]
> It seems to share the same name as this dataset: https://openreview.net/forum?id=JENSKEEzsoU

you're right i can be confusing. I'll add the organization/research group for clarity: `covid_qa_castorini`

**Comment by lhoestq** [comment]
Thanks for avoiding the name collision !

File: datasets/covid_qa_castorini/covid_qa_castorini.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+_HOMEPAGE = "http://covidqa.ai"
+
+_LICENSE = "Apache License 2.0"
+
+_URL = "https://raw.githubusercontent.com/castorini/pygaggle/master/data/"
+_URLs = {"covid_qa_castorini": _URL + "kaggle-lit-review-0.2.json"}
+
+
+class CovidQA(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/covid_qa_castorini/covid_qa_castorini.py:47]:
```suggestion
class CovidQaCastorini(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `covid_qa_castorini`

File: datasets/covid_qa_castorini/dataset_infos.json [hunk 2]
```diff
@@ -0,0 +1 @@
... year={2020}\n}\n", "homepage": "http://covidqa.ai", "license": "Apache License 2.0", "features": {"category_name": {"dtype": "string", "id": null, "_type": "Value"}, "question_query": {"dtype": "string", "id": null, "_type": "Value"}, "keyword_query": {"dtype": "string", "id": null, "_type": "Value"}, "answers": {"feature": {"id": {"dtype": "string", "id": null, "_type": "Value"}, "title": {"dtype": "string", "id": null, "_type": "Value"}, "exact_answer": {"dtype": "string", "id": null, "_type": "Value"}}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "covid_qa", "config_name": "covid_qa_castorini", "version": {"version_str": "0.2.0", "description": null, "major": 0, "minor": 2, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 33757, "num_examples": 27, "dataset_name": "covid_qa"}}, "download_checksums": {"https://raw.githubusercontent.com/castorini/pygaggle/master/data/kaggle-lit-review-0.2.json": {"num_bytes": 51438, "checksum": "b998dee956c4592a63828c628d1a369e6a81b8527e384a9d3448f417008080fb"}}, "download_size": 51438, "post_processing_size": null, "dataset_size": 33757, "size_in_bytes": 85195}}
```
Comment (lhoestq) [inline datasets/covid_qa_castorini/dataset_infos.json:1]:
```suggestion
{"covid_qa_deepset": {"description": "COVID-QA is a Question Answering dataset consisting of 2,019 question/answer pairs annotated by volunteer biomedical experts on scientific articles related to COVID-19.\n", "citation": "@inproceedings{moller2020covid,\n  title={COVID-QA: A Question Answering Dataset for COVID-19},\n  author={M{\"o}ller, Timo and Reina, Anthony and Jayakumar, Raghavan and Pietsch, Malte},\n  booktitle={Proceedings of the 1st Workshop on NLP for COVID-19 at ACL 2020},\n  year={2020}\n}\n", "homepage": "https://github.com/deepset-ai/COVID-QA", "license": "Apache License 2.0", "features": {"document_id": {"dtype": "int32", "id": null, "_type": "Value"}, "context": {"dtype": "string", "id": null, "_type": "Value"}, "question": {"dtype": "string", "id": null, "_type": "Value"}, "is_impossible": {"dtype": "bool", "id": null, "_type": "Value"}, "id": {"dtype": "int32", "id": null, "_type": "Value"}, "answers": {"feature": {"text": {"dtype": "string", "id": null, "_type": "Value"}, "answer_start": {"dtype": "int32", "id": null, "_type": "Value"}}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "covid_qa_deepset", "config_name": "covid_qa_deepset", "version": {"version_str": "1.0.0", "description": null, "major": 1, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 65151262, "num_examples": 2019, "dataset_name": "covid_qa_deepset"}}, "download_checksums": {"https://raw.githubusercontent.com/deepset-ai/COVID-QA/master/data/question-answering/COVID-QA.json": {"num_bytes": 4418117, "checksum": "291abf17f4bc2bd343838fd8ef5debb6278bbbb61b262db1f1bd58048fff76b9"}}, "download_size": 4418117, "post_processing_size": null, "dataset_size": 65151262, "size_in_bytes": 69569379}, "covidqa": {"description": "CovidQA is the beginnings of a question answering dataset specifically designed for COVID-19, built by hand from knowledge gathered from Kaggle's COVID-19 Open Research Dataset Challenge.\n", "citation": "@article{tang2020rapidly,\n  title={Rapidly Bootstrapping a Question Answering Dataset for COVID-19},\n  author={Tang, Raphael and Nogueira, Rodrigo and Zhang, Edwin and Gupta, Nikhil and Cam, Phuong and Cho, Kyunghyun and Lin, Jimmy},\n  journal={arXiv preprint arXiv:2004.11339},\n  year={2020}\n}\n", "homepage": "http://covidqa.ai", "license": "Apache License 2.0", "features": {"category_name": {"dtype": "string", "id": null, "_type": "Value"}, "question_query": {"dtype": "string", "id": null, "_type": "Value"}, "keyword_query": {"dtype": "string", "id": null, "_type": "Value"}, "answers": {"feature": {"id": {"dtype": "string", "id": null, "_type": "Value"}, "title": {"dtype": "string", "id": null, "_type": "Value"}, "exact_answer": {"dtype": "string", "id": null, "_type": "Value"}}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "covid_qa_castorini", "config_name": "covidqa", "version": {"version_str": "0.2.0", "description": null, "major": 0, "minor": 2, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 33757, "num_examples": 27, "dataset_name": "covid_qa_castorini"}}, "download_checksums": {"https://raw.githubusercontent.com/castorini/pygaggle/master/data/kaggle-lit-review-0.2.json": {"num_bytes": 51438, "checksum": "b998dee956c4592a63828c628d1a369e6a81b8527e384a9d3448f417008080fb"}}, "download_size": 51438, "post_processing_size": null, "dataset_size": 33757, "size_in_bytes": 85195}, "covid_qa_castorini": {"description": "CovidQA is the beginnings of a question answering dataset specifically designed for COVID-19, built by hand from knowledge gathered from Kaggle's COVID-19 Open Research Dataset Challenge.\n", "citation": "@article{tang2020rapidly,\n  title={Rapidly Bootstrapping a Question Answering Dataset for COVID-19},\n  author={Tang, Raphael and Nogueira, Rodrigo and Zhang, Edwin and Gupta, Nikhil and Cam, Phuong and Cho, Kyunghyun and Lin, Jimmy},\n  journal={arXiv preprint arXiv:2004.11339},\n  year={2020}\n}\n", "homepage": "http://covidqa.ai", "license": "Apache License 2.0", "features": {"category_name": {"dtype": "string", "id": null, "_type": "Value"}, "question_query": {"dtype": "string", "id": null, "_type": "Value"}, "keyword_query": {"dtype": "string", "id": null, "_type": "Value"}, "answers": {"feature": {"id": {"dtype": "string", "id": null, "_type": "Value"}, "title": {"dtype": "string", "id": null, "_type": "Value"}, "exact_answer": {"dtype": "string", "id": null, "_type": "Value"}}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "covid_qa_castorini", "config_name": "covid_qa_castorini", "version": {"version_str": "0.2.0", "description": null, "major": 0, "minor": 2, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 33757, "num_examples": 27, "dataset_name": "covid_qa_castorini"}}, "download_checksums": {"https://raw.githubusercontent.com/castorini/pygaggle/master/data/kaggle-lit-review-0.2.json": {"num_bytes": 51438, "checksum": "b998dee956c4592a63828c628d1a369e6a81b8527e384a9d3448f417008080fb"}}, "download_size": 51438, "post_processing_size": null, "dataset_size": 33757, "size_in_bytes": 85195}}
```

`covid_qa` -> `covid_qa_castorini`

**Review by lhoestq — APPROVED** [review]
Awesome thanks :)

**Pull request merged**